src/libostree/ostree-sysroot.c

```diff
@@ -729,6 +729,20 @@ ostree_sysroot_load (OstreeSysroot  *self,
   return ostree_sysroot_load_if_changed (self, NULL, cancellable, error);
 }
 
+static gboolean
+ensure_repo_opened (OstreeSysroot  *self,
+                    GError        **error)
+{
+  if (self->repo_opened)
+    return TRUE;
+  if (!ostree_repo_open (self->repo, NULL, error))
+    return FALSE;
+  /* And flag it as a system repo */
+  self->repo->is_system = TRUE;
```

Comment:
Yeah, I removed this.  It'll come back though in the later patches when this bit moves around.